examples/pkg_depend/pkg_depend/__init__.py

```diff
@@ -1,9 +1,6 @@
 try:
     from autover.version import Version
-    versionobj = Version(release=None, fpath=__file__,
-                         archive_commit="$Format:%h$", reponame="pkg_depend")
-    __version__ = str(versionobj)
+    __version__ = str(Version(fpath=__file__,archive_commit="$Format:%h$",reponame="pkg_depend"))
 except:
     import os, json
-    __version__ = json.load(open(os.path.join(os.path.split(__file__)[0],
-                                              '.version'), 'r'))['version_string']
+    __version__ = json.load(open(os.path.join(os.path.dirname(__file__),'.version'),'r'))['version_string']
```

Comment:
If project depends on autover, doesn't that make this except unnecessary?

Reply:
You are right that it isn't strictly necessary (in this *particular* test!) so I'll leave it up to you whether you want to remove it or not. Making things shorter is nice but so is using the same thing everywhere so you don't need to think too much about it.

Reply:
At any rate, it has no effect on the functionality here so if you decide you simplify it, it can be done in a separate PR.